### DISCUSSION TITLE
File

### DISCUSSION THREAD

**Discussion by hamidullaorifov** [python-telegram-bot/python-telegram-bot#3170] [Q&A]
how can i get large files?

**Comment by harshil21** [comment]
If by "getting" large files you mean download, see https://github.com/python-telegram-bot/python-telegram-bot/wiki/Code-snippets#downloading-a-file.

If you want to upload large files, you should host your own local Bot API server: https://github.com/python-telegram-bot/python-telegram-bot/wiki/Local-Bot-API-Server
